microservices/cart/config.yml

```diff
@@ -0,0 +1,74 @@
+service:
+  name: carts
+  injections:
+    postgresql_pool: minos.common.PostgreSqlPool
+    command_broker: minos.networks.CommandBroker
+    command_reply_broker: minos.networks.CommandReplyBroker
+    event_broker: minos.networks.EventBroker
+    repository: minos.common.PostgreSqlRepository
+    reply_pool: minos.networks.ReplyHandlerPool
+    saga_manager: minos.saga.SagaManager
+    snapshot: minos.common.PostgreSqlSnapshot
+    discovery: minos.networks.DiscoveryConnector
+    cart_repository: src.CartRepository
+  services:
+    - minos.networks.CommandConsumerService
+    - minos.networks.CommandHandlerService
+    - minos.networks.CommandReplyConsumerService
+    - minos.networks.CommandReplyHandlerService
+    - minos.networks.EventConsumerService
+    - minos.networks.EventHandlerService
+    - minos.networks.RestService
+    - minos.networks.SnapshotService
+    - minos.networks.ProducerService
+rest:
+  host: 0.0.0.0
+  port: 8086
```

Comment:
This port collides with the `user` microservice, which would generates troubles during development. Could you pick another one? I think the `8087` is free.

Reply:
Done!